### PR TITLE
Closes #307 - Create coursePrice table, Entity, DAO

### LINKED ISSUE #307: Create coursePrice table, Entity, DAO

### PR DESCRIPTION
**Pull request by maxturko**

### CODE REVIEW

File: src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE course_price (
+price_id bigint not null auto_increment,
+price integer,
+course_id bigint,
+primary key (price_id)
+);
+alter table course_price add constraint FK4leaja1jtefsdfaiqqk4yuxna foreign key (course_id) references course (course_id);
```
Comment (Maks-Chornyi) [inline src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql]: add one more line, please

Comment (maxturko) [reply src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql]: done

Comment (antonbakalets) [reply src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql]: What does FK4leaja1jtefsdfaiqqk4yuxna mean?

I suggest to rename it to COURSE_PRICE_COURSE_FK

**Review by antonbakalets — COMMENTED** [review]
I believe you also have to add field to Cource referencing CourcePrice with NULLs allowed and with a corresponding Foreign Key. Make sure the application is still starting.

File: src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql (hunk 1 above)
Comment (maxturko) [reply src/main/resources/db/migration/V1.8__Create_CoursePrice_table.sql]: Fixed

**Review by OleksandrMazur — APPROVED** [review]

File: src/main/java/com/softserve/academy/spaced/repetition/domain/CoursePrice.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+@Table(name = "course_price")
+public class CoursePrice {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "price_id")
+    private long id;
+
+    @Column(name = "price")
+    private long price;
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/domain/CoursePrice.java]: Int

Comment (maxturko) [reply src/main/java/com/softserve/academy/spaced/repetition/domain/CoursePrice.java]: done

**Review by antonbakalets — APPROVED** [review]

**Review by Maks-Chornyi — APPROVED** [review]

**Review by volodymyrglynka — APPROVED** [review]

**Pull request merged**